src/components/vacation-requests-table/vacation-requests-table.tsx

```diff
@@ -51,14 +52,16 @@ interface Props {
  * @param props component properties
  */
 const VacationRequestsTable = ({
+  isUpcoming,
+  toggleIsUpcoming,
   deleteVacationRequests,
   createVacationRequest,
   updateVacationRequest,
   updateVacationRequestStatuses,
   loading
 }: Props) => {
   const adminMode = UserRoleUtils.adminMode();
-  const vacationRequests = useAtomValue(adminMode ? allVacationRequestsAtom : vacationRequestsAtom);
+  const vacationRequests = useAtomValue(displayedVacationRequestsAtom);
```

Comment:
Could this be related to the basic/ admin sync issue you mentioned here. As I understand it regardless of the displayed vacations in the "basic" view shouldn't the admin mode still see all users holidays? Of course it would also be good for admins to be able to toggle past/ upcoming also

Reply:
Admin does see all vacations and is able to toggle between upcoming and past in current implementation.